scripts/qa_crowbarsetup.sh

```diff
@@ -4480,8 +4480,13 @@ function onadmin_crowbarrestore()
 function onadmin_crowbar_nodeupgrade()
 {
     local endpoint
+    local http_code
     for endpoint in services backup nodes; do
-        safely curl --silent -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/$endpoint.json
+        http_code=`curl -X POST -s -o crowbar-node-upgrade.txt -w '%{http_code}' $crowbar_api_digest $crowbar_api/installer/upgrade/$endpoint.json`
+        if ! [[ $http_code =~ [23].. ]] ; then
+            cat crowbar-node-upgrade.txt
+            complain 36 "Request to $crowbar_api/installer/upgrade/$endpoint.json returned $http_code"
+        fi
```

Comment:
We are using this logic (checking the http return code) in 2 more places. Want to refactor this into its own function? That would also help @dirkmueller with his reading the code ;)